test/packages/knowledge_base/kibana/kb_entry/foo/index-mapping.json

```diff
@@ -0,0 +1,42 @@
+{
+  "dynamic": "strict",
+  "properties": {
```

Comment:
Are these properties going to be the same in all knowledge bases?

Reply:
Related to https://github.com/elastic/package-spec/pull/807#discussion_r1775216062 - they won't, we want to stay open to the format. As long as we know which fields are searchable (semantic/syntactic) our retrieval logic will manage.